src/encoded/static/components/experiment.js

```diff
@@ -927,16 +927,7 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
             />
 
             {isFunctionalExperiment && plasmidMapDocs.length > 0 ?
-                <div>
-                    <Panel>
-                        <PanelHeading>
-                            <h4>Plasmid maps</h4>
-                        </PanelHeading>
-                        <PanelBody addClasses="panel-body-doc-interior">
-                            <DocumentsSubpanels documentSpec={{ documents: plasmidMapDocs }} />
-                        </PanelBody>
-                    </Panel>
-                </div>
+                <DocumentsPanelReq documents={plasmidMapDocs} title="Plasmid Maps" />
```

Comment:
Is there a reason you changed the title from "Plasmid maps" to "Plasmid Maps"? I don't know if we have a definite convention about this but I think in general we have the first word capitalized and second words uncapitalized. Like we have "Isogenic replicates" instead of "Isogenic Replicates".

Reply:
Using this component instead of `<DocumentsSubpanels>` is much better.